### PR TITLE
Add test to address a PROTOTYPE comment.

### PR DESCRIPTION
**Pull request by AlekseyTs**

### CODE REVIEW

**Comment by AlekseyTs** [comment]
@333fred, @mavasani, @cston, @dotnet/roslyn-compiler, @dotnet/analyzer-ioperation Please review

**Review by mavasani — APPROVED** [review]

**Review by cston — APPROVED** [review]

File: src/Test/Utilities/Portable/Compilation/TestOperationVisitor.cs [hunk 1]
```diff
@@ -1238,6 +1238,13 @@ public override void VisitFlowCapture(IFlowCaptureOperation operation)
                 case OperationKind.DefaultValue:
                 case OperationKind.FlowAnonymousFunction: // must be an error case like in Microsoft.CodeAnalysis.CSharp.UnitTests.ConditionalOperatorTests.TestBothUntyped unit-test
                     break;
+
+                case OperationKind.OmittedArgument:
+                case OperationKind.DeclarationExpression:
+                case OperationKind.Discard:
+                    Assert.False(true);
```
Comment (333fred) [inline src/Test/Utilities/Portable/Compilation/TestOperationVisitor.cs]: Consider adding a message for why the test failed.

**Review by 333fred — APPROVED** [review]
LGTM (iteration 1). Message can be added in a follow up.

**Comment by etbyrd** [comment]
The integration test failure looked like a remoting issue.

**Pull request merged**